libsrc/telestrat/gotox.s

```diff
@@ -3,13 +3,8 @@
 ;
     .export    _gotox
 
-    .import    popa
-
-    .importzp  sp
-
     .include   "telestrat.inc"
 
-
 .proc _gotox
    sta    SCRX
    rts
```

Comment:
I think that `gotox()` needs to "reset" the old colors, as `gotoxy()` does.  Consider this program flow:
- A row is divided into three areas.
- Some text is printed in the middle area, in one color.
- Then, some text is printed in the left area, in a different color.
- Then, some text is printed in the right area, in the same color as the left area.
- `gotox()` is used to move between the areas.

Currently, no attribute is put into the right area; but, that allows the middle color "to bleed" onto the right-side text!  If 'gotox()' sets the old colors to `$FF`, then the right area will get attributes.  It always will show its desired colors.

Reply:
Fixed